index.html

```diff
@@ -0,0 +1,241 @@
+      </div>
+    </div>
+  </header>
+  <main>
+    <section class="banner">
+      <div class="container flex-box">
+        <div class="panel-left">
+          <h2>Welcome to</h2><h5>Gardener</h5><h2>Landscaping Company.</h2>
+          <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+          Lorem Ipsum has been the industry's standard dummy text ever since the 
+          1500s, when an unknown printer took a gallery of type and scrambled it to make.</p>
+          <a href="contact.html" class="btn btn-primary">Contact Us</a>
+        </div>
+        <div class="panel-right">
+          <img src="images\img-1.jpg" alt="Home">
+        </div>
+      </div> 
+    </section> 
+    <div class="container">
+      <section class="block block-center">
+        <div class="title-area">
+          <h5>Offer</h5>
+          <h3>What We Offer</h3>
+        </div>
+        <ul class="listing-style listing-style-one flex-box">
+          <li>
+            <img src="images\ico-1.png" alt="Planning">
+            <h4>Planning Your Garden</h4>
+            <p>Contrary to popular belief, Lorem Ipsum is not
+              simply random text. It has roots in a piece of
+              classical Latin literature from 45 BC</p>
+          </li>
+          <li>
+            <img src="images\ico-2.png" alt="Designing">
+            <h4>Designing Landscape</h4>
+            <p>Contrary to popular belief, Lorem Ipsum is not
+              simply random text. It has roots in a piece of
+              classical Latin literature from 45 BC</p>
+          </li>
+          <li>
+            <img src="images\ico-3.png" alt="Lawn">
+            <h4>Driveway & Lawn</h4>
+            <p>Contrary to popular belief, Lorem Ipsum is not
+              simply random text. It has roots in a piece of
+              classical Latin literature from 45 BC</p>
+          </li>
+        </ul>
+      </section>
+      <section class="block flex-box col-holder">
+        <div class="block-panel-left">
+          <img src="images\img-2.jpg" alt="About us">
+        </div>
+        <div class="block-panel-right ">
+          <h5>About Us</h5>
+          <h3>Welcome To Our<br> Gardening</h3>
+          <p>It is a long established fact that a reader will be distracted by the readable
+            content of a page when looking at its layout. Many desktop publishing
+            packages and web page editors now use Lorem Ipsum as their default model
+            text, and a search for 'lorem ipsum' will uncover many web sites still in their 
+            infancy</p>
+          <a href="Read" class="btn btn-secondary">Read More</a>  
+        </div>
+      </section>
+      <section class="block">
+        <div class="title-area block-center">
+          <h5>Services</h5>
+          <h3>Landscape Solutions</h3>
+          <ul class="listing-style listing-style-two flex-box">
+            <li>
+              <img src="images\ico-4.png" alt="Renovation">
```

Comment:
Alignment issue for mobile. For the mobile view, icon should be center aligned. refer to design.